srcpkgs/python3-tomlkit/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-tomlkit'
+pkgname=python3-tomlkit
+version=0.11.0
+revision=1
+wrksrc="${pkgname/python3-/}-${version}"
+build_style=python3-pep517
+hostmakedepends="python3-poetry-core"
+depends="python3"
+short_desc="Style-preserving TOML library for Python"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT"
+homepage="https://github.com/sdispater/tomlkit"
+changelog="https://github.com/sdispater/tomlkit/raw/master/CHANGELOG.md"
+distfiles="${PYPI_SITE}/t/${pkgname/python3-/}/${pkgname/python3-/}-${version}.tar.gz"
+checksum=71ceb10c0eefd8b8f11fe34e8a51ad07812cb1dc3de23247425fbc9ddc47b9dd
+#Requires a git submodule for running tests
+make_check=no
```

Comment:
Could you explain why you think this package requires a git submodule for running tests? I had the package ready for a pylint update that I didn't merge yet, so I pushed my template with tests working: 72d051fb7312c60bbb05fc96f9df4662d99e7c90.

Reply:
It's what upstream claims, see https://github.com/sdispater/tomlkit/#running-tests

I've just seen your merged template in the notifications, and am rebasing. Happy  to hear that apparently we can run the tests there.

Reply:
But the PyPI tarballs have that submodule bundled inside. It's always good to try to run tests before looking upstream.